### PR TITLE
add translations/wallabag_user.de.yml (fixes #2673)

### LINKED ISSUE #2673: Translate wallabag_user.en.yml into German
wallabag_user.de.yml is missing: https://github.com/wallabag/wallabag/tree/2.2/src/Wallabag/UserBundle/Resources/translations

### PR DESCRIPTION
**Pull request by Strubbl**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/
| New feature?  | /no
| BC breaks?    | /no
| Deprecations? | /no
| Tests pass?   | yes/
| Documentation | /no
| Translation   | yes/
| Fixed tickets | #2673
| License       | MIT

add missing german translation

### CODE REVIEW

File: src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Two factor mail
+auth_code:
+    on: 'an'
+    mailer:
+        subject: 'Wallabag Authentifizierungcode'
```
Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml]: wallabag instead of Wallabag ;-)

Comment (Strubbl) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml]: yes. this is also wrong in the english file.

Comment (nicosomb) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml]: not on the 2.2 branch https://github.com/wallabag/wallabag/blob/2.2/src/Wallabag/UserBundle/Resources/translations/wallabag_user.en.yml ;-)

Comment (Strubbl) [inline src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml]: So is it correct that my PR wants to merge to master? Or should i point it to the 2.2 branch?

**Comment by nicosomb** [comment]
Thank you!

**Pull request merged**